ci/common/build.bash

```diff
@@ -334,6 +365,19 @@ log "Test Thrust and CUB..."
 )
 test_status=$?
 
+################################################################################
+# COMPILATION STATS
+################################################################################
+
+if [[ -n "${ENABLE_SCCACHE}" ]]; then
+  # Get sccache stats after the compile is completed
+  COMPILE_REQUESTS=$(sccache -s | grep "Compile requests \+ [0-9]\+$" | awk '{ print $NF }')
+  CACHE_HITS=$(sccache -s | grep "Cache hits \+ [0-9]\+$" | awk '{ print $NF }')
+  HIT_RATE=$(echo - | awk "{printf \"%.2f\n\", $CACHE_HITS / $COMPILE_REQUESTS * 100}")
+  log "sccache stats (${HIT_RATE}% hit):"
```

Comment:
This looks like it was copy/pasted from a RAPIDS repo. It can be removed if desired. For reference, the `sccache -s` command below this code hunk outputs details that look like this, which seems sufficient:


```
Compile requests                      614
Compile requests executed             614
Cache hits                             73
Cache hits (C/C++)                     44
Cache hits (CUDA)                      29
Cache misses                          541
Cache misses (C/C++)                  243
Cache misses (CUDA)                   298
Cache timeouts                          0
Cache read errors                       0
Forced recaches                         0
Cache write errors                      0
Compilation failures                    0
Cache errors                            0
Non-cacheable compilations              0
Non-cacheable calls                     0
Non-compilation calls                   0
Unsupported compiler calls              0
Average cache write                 0.414 s
Average cache read miss           123.783 s
Average cache read hit              0.180 s
Failed distributed compilations         0
Cache location                  S3, bucket: Bucket(name=rapids-sccache, base_url=http://rapids-sccache.s3-us-west-2.amazonaws.com/)
```

Reply:
Sounds good. I like having the hit rate percentage in the log message along with the details, so we can keep both.